src/components/MainView/DataView/DataView.js

```diff
@@ -248,9 +248,13 @@ export const DataView = injector(
           },
         },
         {
-          resolver: (col) => col.type === "Image",
+          resolver: (col) => col.type === "Image" && getRoot(col.original).SDK.type !== 'DE',
```

Comment:
what happens if col.original is empty, or produces a non existent return value from getRoot? This will explode since it is directly accessing SDK and type.

Reply:
Fair point, I'll make this a little more null safe